app/jobs/identities_backfill_job.rb

```diff
@@ -0,0 +1,58 @@
+class IdentitiesBackfillJob < ApplicationJob
+  # This is a short-term solution to backfill data requiring a table scan.
+  # This job can be deleted once it's done.
+
+  queue_as :low
+
+  # Let's give us the option to fine-tune this on the fly
+  BATCH_SIZE_KEY = 'IdentitiesBackfillJob.batch_size'.freeze
+  SLICE_SIZE_KEY = 'IdentitiesBackfillJob.slice_size'.freeze
+  CACHE_KEY = 'IdentitiesBackfillJob.position'.freeze
+
+  def perform
+    start_time = Time.zone.now
+    max_id = ServiceProviderIdentity.last.id
+
```

Comment:
I think we need some sort of early-abort here 

```suggestion
    if position > max_id
      logger.info "backfill reached end, skipping. position=#{position} max_id=#{max_id}"
      return
    end
```